### DISCUSSION TITLE
Multiple databases

### DISCUSSION THREAD

**Discussion by ArtemisGraphic** [pocketbase/pocketbase#4941] [Q&A]
Hello,
is it possible to have multiple databases? for example if my platform has many sellers, each seller will have one db with collections in it. But I see PB is just one file. How can I manage this structure?
Thank you

**Comment by RepComm** [comment]
Two options I could see are:
- Use one db, and use a relation to specify what 'seller' is responsible for your collections as a field
- Start another pocketbase instance from another directory so that it generates it's own pb_data directory in it. Manages separate instances where you can't predict the final number (assuming you can have unlimited sellers in your business model) could get intense to manage.

pocketbase itself being one file is simply the tool itself, pocketbase stores your actual database in the pb_data directory under whatever directory you started it from.

```bash
# change directories to desktop folder
cd ~/Desktop/

# start pocketbase listening on all interfaces on port 8080
pocketbase serve --http=0.0.0.0:8080

# list files in current directory
ls

# pb_data folder should exist, containing your db files
> ... pb_data ...
